workspaces/ui-v2/src/optic-components/diffs/render/AskForCommitMessage.tsx

```diff
@@ -57,28 +57,32 @@ export default function AskForCommitMessageDiffPage(props: {
 
   const handleSave = async (commitMessage: string) => {
     const commands = context.simulatedCommands;
-    const {
-      data: {
+    try {
```

Comment:
Added very rudimentary error catch blocks - I think we will want to have some generic error handlers (and perhaps toasts or something) but it doesn't look like we have any existing patterns.

Added this in here since there's no type safe way to swallow exceptions from the spectacleCommand function - but this is also unideal